openshift-ci/Dockerfile.tools

```diff
@@ -24,16 +22,13 @@ RUN yum install epel-release -y \
     python36-virtualenv \
     && yum clean all
 
-# install dep
-RUN mkdir -p $GOPATH/bin && chmod a+rwx $GOPATH \
-    && curl -L -s https://github.com/golang/dep/releases/download/v0.5.1/dep-linux-amd64 -o dep \
-    && echo "7479cca72da0596bb3c23094d363ea32b7336daa5473fa785a2099be28ecd0e3  dep" > dep-linux-amd64.sha256 \
-    && sha256sum -c dep-linux-amd64.sha256 \
-    && rm dep-linux-amd64.sha256 \
-    && chmod +x ./dep \
-    && mv dep $GOPATH/bin/dep
+RUN yum -y install  https://centos7.iuscommunity.org/ius-release.rpm \
```

Comment:
what about moving to centos:8 instead?

Reply:
i didn't find one in `openshift` namespace in ci-cluster
https://github.com/openshift/release/blob/97ac15779bdfd82414b418166347018eecefe748/ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml#L6

i can try, but it will take more time. shall we get it working with this and move to centos:8 after the demo on wednesday

Reply:
@chmouel im pushing a patch (cnetos:8) to openshift/release

Reply:
ah yeah right you can only do from this side,